app/views/admin/redeem_requests/index.html.haml

```diff
@@ -2,22 +2,22 @@
   %h1
     Redeem Requests
     %span.badge $ #{amount_for_store}
-    %span.dropdown
-      %button.btn.btn-default.dropdown-toggle#store{"data-toggle" => "dropdown", type: "button"} By Store
-      %ul.dropdown-menu
-        - REDEEM['amazon_stores'].each do |store|
-          %li
-            =link_to "#{store} $#{amount_for_store(store)}", admin_redeem_requests_path(store: store), class: 'closeDropdown', remote: true
-        %li
-          =link_to "All", admin_redeem_requests_path, class: 'closeDropdown', remote: true
+    %br
+    %br
+  %label.slide
+  = link_to download_admin_redeem_requests_path(:status => "true"), class: "btn btn-primary glyphicon glyphicon-download", style: 'margin-left: 5px;' do
+    %b Completed
+  = link_to download_admin_redeem_requests_path(:status => "false"), class: "btn btn-primary glyphicon glyphicon-download" do
+    %b Active
 
-    %label.slide
-      %input.check#redeem{checked: 'checked', name: 'status', type: 'checkbox', data: {toggle: 'toggle', width: '100', height: '40', on: 'Status Open', off: 'Status Close'}}
-      #console-event
-    = link_to download_admin_redeem_requests_path(:status => "true"), class: "btn btn-primary glyphicon glyphicon-download" do
-      %b Completed
-    = link_to download_admin_redeem_requests_path(:status => "false"), class: "btn btn-primary glyphicon glyphicon-download" do
-      %b Active
+  = form_for(RedeemRequest.new, url: admin_redeem_requests_path, method: 'get', remote: true) do |f|
+    - stores = REDEEM['amazon_stores'].map { |store| ["#{store} #{amount_for_store(store)}", store] }
+
+    = f.select :store, stores.unshift(['All']), {}, {onchange: "$(this.form).submit();", class: "btn btn-primary"}
```

Comment:
Dont add inline javascript. Create a seperate javascript file, redeem_request.js and there all the javascript